fabstract/Hales_T_et_al_Kepler_conj/fabstract.lean

```diff
@@ -61,14 +62,34 @@ unfinished Kepler_conjecture :
 { description := "Proof of Kepler conjecture",
   references := [cite.Item cite.Ibidem "Theorem X.Y"] }
 
-open result
-
 def fabstract : meta_data := {
   description := "This article announces the formal proof of the Kepler conjecture on dense sphere packings in a combination of the HOL Light and Isabelle/HOL proof assistants.  It represents the primary result of the now completed Flyspeck project.",
-  authors := ["Thomas Hales", "Mark Adams", "Gertrud Bauer", "Tat Dat Dang", "John Harrison", "Le Truong Hoang", "Cezary Kaliszyk", "Victor Magron", "Sean McLaughlin", "Tat Thang Nguyen", "Quang Truong Nguyen", "Tobias Nipkow", "Steven Obua", "Joseph Pleso", "Jason Rute", "Alexey Solovyev", "Thi Hoai An Ta", "Nam Trung Tran", "Thi Diep Trieu", "Josef Urban", "Ky Vu", "Roland Zumkeller"],
+  authors := [
+    ({name := "Thomas Hales"} : author), -- TODO why do I have cast the type here but not in other fabstracts?
```

Comment:
I think one of those `open` statements must be messing with the type inference. In other fabstracts no such type cast was needed. And I tried `{| author, name := "Thomas Hales" |}` but it did not work. These complications make me doubt existence of Thomas Hales once again.